runtime-modules/storage/src/data_object_storage_registry.rs

```diff
@@ -121,6 +125,34 @@ decl_storage! {
         /// Keeps a list of storage relationships per content id.
         pub RelationshipsByContentId get(relationships_by_content_id): map T::ContentId => Vec<T::DataObjectStorageRelationshipId>;
     }
+    add_extra_genesis {
```

Comment:
This can be moved out into the working-group module itself, and the config settings can be set on each instance in the runtime configuration. see my previous comment.

Reply:
Done.